examples/accordion/accordion.html

```diff
@@ -49,19 +49,17 @@ <h2 id="ex_label">Example</h2>
 
           Ex:
             <dl id="accordionGroup" role="presentation" class="Accordion" data-allow-multiple>
-
             <dl id="accordionGroup" role="presentation" class="Accordion" data-allow-toggle>
-
         -->
-        <dl id="accordionGroup" role="presentation" class="Accordion">
-          <dt role="heading" aria-level="3">
+        <div id="accordionGroup" role="presentation" class="Accordion">
```

Comment:
We do not need role presentation on a div (yes, sometimes if it has a tooltip you do that to avoid issues on Mac, but don't have that situation here)